scripts/bundlesize/bundlesize.config.json

```
@@ -2,7 +2,7 @@
     "files": [
         {
             "path": "packages/@lwc/engine-dom/dist/index.js",
-            "maxSize": "24.72KB"
+            "maxSize": "24.75KB"
```

```suggestion
            "maxSize": "25KB"
```
Changing this every PR is dumb. Round up!